Xwt.XamMac/Xwt.Mac/Util.cs

```diff
@@ -240,6 +240,21 @@ public static NSBitmapImageFileType ToMacFileType (this ImageFileType type)
 			}
 		}
 
+		public static bool TriggerContextMenu (this NSEvent theEvent)
```

Comment:
Should we call this `TriggersContextMenu` to match the extension method in the Gtk backend?